.env

```diff
@@ -18,6 +18,11 @@ SERPER_API_KEY=#your serper.dev api key here
 SERPAPI_KEY=#your serpapi key here
 SERPSTACK_API_KEY=#your serpstack api key here
 USE_LOCAL_WEBSEARCH=#set to true to parse google results yourself, overrides other API keys
+USE_LOCAL_SEARXNG_URL=#your SearXNG URL eg https://etsi.me (ensure json enabled on instance settings.yml)
+
+#SearXNG configuration for public instances (or use default settings in self hosted instances).
+SEARXNG_CATEGORIES=#comma seperated general,images,videos,news,map,music,it,science,files,social_media (or undefined for instance defaults).
```

Comment:
Is the customization needed or can we have some sane defaults that are used across the app ?

I feel like it's a lot of new env variables that most people won't need. Same for `SEARXNG_ENGINES` but i'm not super familiar with the project so please tell me if they're needed.

Reply:
I feel like given the parameters are just injected into the URL as query parameters, could we rewrite this so that `SEARXNG_URL` already contains the query parameters for engines, categories, etc. if people want to set them ?

Would require a bit of URL parsing logic to correctly append the query, but it would make it possible to have only one env variable for this. what do you think?

Reply:
I'll try to get these into a single string.

Reply:
> Is the customization needed or can we have some sane defaults that are used across the app ?
> 
> I feel like it's a lot of new env variables that most people won't need. Same for `SEARXNG_ENGINES` but i'm not super familiar with the project so please tell me if they're needed.

I think engines is important but could do away with categories (as its  engine grouping).